content/source/docs/enterprise/private/minio-setup-guide.html.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: "enterprise2"
+page_title: "Private Terraform Enterprise Installation (Installer) - Minio Setup Guide"
+sidebar_current: "docs-enterprise2-private-installer-minio"
+description: |-
+  This document provides an overview for setting up minio for external object storage for HashiCop Private Terraform Enterprise (PTFE).
+---
+
+## Introduction
+
+This document provides an overview for setting up minio for external object storage for HashiCop Private Terraform Enterprise (PTFE).
+
+When configured to use external services, PTFE must be connected to a storage service to persist workspace state and other file-based data.  Native support exists for Azure Blob Storage, Amazon S3, and services that are API-compatible with Amazon S3.  If you are not using Azure or a cloud provider with an S3-compatible service, or you are running PTFE in an environment without a storage service, it may be possible to use [minio](https://minio.io) instead.  Minio is an Amazon S3-compatible object storage server that can use mounted filesystems or a variety of other cloud storage solutions to persist data.  HashiCorp may add support in the future for other cloud storage solutions to meet customer requests.
```

Comment:
I'd suggest we don't add promises of future support to the docs - it leads to lots of "but when?!" tickets.

Reply:
ok, that's fair.